lib/common_test/doc/src/ct_hooks_chapter.xml

```diff
@@ -454,19 +454,21 @@
 
  %% Called before each init_per_testcase.
  pre_init_per_testcase(Suite,TC,Config,State) ->
```

Comment:
Example still generates warnings for multiple unused variables (e.g. `Suite`), which might be confusing for beginners.
Should unused variable names be prefixed with '_' to silence the unwanted output? What do you think?

Reply:
Hmm, OK I guess we had different linters configured. I can see what it looks like if we fix that as well.

But I think using a deprecated function is a bigger, and "new" issue.

The unused variables were there before and are the same issue now as before, whereas the problem with the deprecated function was introduced (well, kind of) recently by OTP evolving.

Reply:
I agree with what you wrote :-)